src/index.js

```diff
@@ -134,5 +134,22 @@ function resizeEvent() {
     resizeSlider();
 }
 
+// -----Instagram Photo-----
+async function getPhoto() {
+    const article = document.querySelectorAll('.article');
+    const data = await fetch('https://www.instagram.com/website_of_the_day/?__a=1')
+        .then(response => response.json())
+        .then(data => data.graphql.user.edge_owner_to_timeline_media.edges);
```

Comment:
There may be an Ajax class somewhere in the boilerplate. It has kind of error catcher and also simple to use.
With it it'd look something like this:
```
const data = await Ajax.get('https://www.instagram.com/website_of_the_day/?__a=1');  
/* I'm not sure about the name */  
const post = data.graphql.user.edge_owner_to_timeline_media.edges;  
```